files/default/sql/ddl/2.1.0__initial_tables.sql

```diff
@@ -1014,21 +1014,39 @@ CREATE TABLE `project` (
   `created` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   `retention_period` date DEFAULT NULL,
   `archived` tinyint(1) DEFAULT '0',
-  `conda` tinyint(1) DEFAULT '0',
   `logs` tinyint(1) DEFAULT '0',
   `deleted` tinyint(1) DEFAULT '0',
-  `python_version` varchar(25) COLLATE latin1_general_cs DEFAULT NULL,
   `description` varchar(2000) COLLATE latin1_general_cs DEFAULT NULL,
   `payment_type` varchar(255) COLLATE latin1_general_cs NOT NULL DEFAULT 'PREPAID',
   `last_quota_update` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   `kafka_max_num_topics` int(11) NOT NULL DEFAULT '100',
   `docker_image` varchar(255) COLLATE latin1_general_cs DEFAULT NULL,
+  `python_env_id` int(11) DEFAULT NULL,
   PRIMARY KEY (`id`),
   UNIQUE KEY `projectname` (`projectname`),
   UNIQUE KEY `inode_pid` (`inode_pid`,`inode_name`,`partition_id`),
   KEY `user_idx` (`username`),
   CONSTRAINT `FK_262_290` FOREIGN KEY (`username`) REFERENCES `users` (`email`) ON DELETE NO ACTION ON UPDATE NO ACTION,
-  CONSTRAINT `FK_149_289` FOREIGN KEY (`inode_pid`,`inode_name`,`partition_id`) REFERENCES `hops`.`hdfs_inodes` (`parent_id`,`name`,`partition_id`) ON DELETE CASCADE ON UPDATE NO ACTION
+  CONSTRAINT `FK_149_289` FOREIGN KEY (`inode_pid`,`inode_name`,`partition_id`) REFERENCES `hops`.`hdfs_inodes` (`parent_id`,`name`,`partition_id`) ON DELETE CASCADE ON UPDATE NO ACTION,
+  CONSTRAINT `FK_362_309` FOREIGN KEY (`python_env_id`) REFERENCES `python_environment` (`id`) ON DELETE SET NULL ON UPDATE NO ACTION
```

Comment:
let's make the constraint names more human-friendly

Reply:
done